crates/solver-types/src/order.rs

```diff
@@ -78,3 +78,48 @@ pub struct FillProof {
 	/// Address of the oracle that attested to the fill.
 	pub oracle_address: String,
 }
+
+/// Order response for API endpoints.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct OrderResponse {
+	/// Unique identifier for this order
+	pub id: String,
+	/// Current order status
+	pub status: DetailedIntentStatus,
+	/// Timestamp when this order was created
+	#[serde(rename = "createdAt")]
+	pub created_at: u64,
+	/// Timestamp when this order was last updated
+	#[serde(rename = "lastUpdated")]
+	pub last_updated: u64,
+	/// Associated quote ID if available
+	#[serde(rename = "quoteId")]
+	pub quote_id: Option<String>,
+	/// Input asset and amount
+	#[serde(rename = "inputAmount")]
+	pub input_amount: AssetAmount,
+	/// Output asset and amount
+	#[serde(rename = "outputAmount")]
+	pub output_amount: AssetAmount,
+	/// Settlement mechanism type
+	#[serde(rename = "settlementType")]
+	pub settlement_type: SettlementType,
+	/// Settlement-specific data
+	#[serde(rename = "settlementData")]
+	pub settlement_data: serde_json::Value,
+	/// Execution details if available
+	#[serde(rename = "executionDetails")]
+	pub execution_details: Option<serde_json::Value>,
```

Comment:
I wonder, what's the purpose of this field? Would it be transaction? Maybe we should call it that?